packages/app/src/components/List/ListEntry.js

```diff
@@ -4,10 +4,11 @@ import PropTypes from 'prop-types'
 import './ListEntry.css'
 import authService from '../../services/authentication'
 
-const token = authService.getToken()
-const isLoggedIn = token !== undefined
 
 const ListEntry = ({ item, onClickRemove, onClickEdit }) => {
+
+  const token = authService.getToken()
+  const isLoggedIn = token !== undefined
```

Comment:
I don't like this spacing.
```suggestion
  const token = authService.getToken()
  const isLoggedIn = token !== undefined

```